### PR TITLE
Trivial doc tidying.

### PR DESCRIPTION
**Pull request by rk295**
I pulled the command line options out of the source and into this doc to
save other people the trouble of digging into the source. Hope nobody
minds!

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2686)

### CODE REVIEW

**Comment by WillemMali** [comment]
Thanks for documenting those flags! I verified that no content is lost in the other parts, and they're all sensible changes.

I would like to ask you to keep the line width <= 80 or <=72 in `dns-controller/README.md`, so the docs are still easily readable from the terminal.

**Comment by rk295** [comment]
Thanks for the review @WillemMali, I've pushed some changes that tidies them up. I've kept `flags.md` to <=80 and `README.md` to <=72.

**Comment by WillemMali** [comment]
Looks good to me! I again verified no content was misplaced.

One Git-related suggestion for the future: you can force-push over the commit in your fork, this is correctly picked up by the CI system. If you end up getting conflicts you can rebase onto master, fix the conflicts there and then force-commit again. This way you always end up with a single commit for the PR.

(Also, I love rebase workflows, I'm so glad I learned how to do that here.)

It's really no problem at all, thanks for the PR! @chrislovecnm could you give this one the green light?

**Comment by rk295** [comment]
@WillemMali wicked, thanks. I'll go read about force-pushing over the existing commit!

**Comment by chrislovecnm** [comment]
@k8s-bot ok to test

**Comment by chrislovecnm** [comment]
Need @justinsb  or @sethpollack to review.  They are far more familiar with the dns controller ;)

File: dns-controller/README.md [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
+* `dns.alpha.kubernetes.io/external` will set up records for accessing 
+  the resource externally
+* `dns.alpha.kubernetes.io/internal` will set up records for accessing 
+  the resource internally
 
 When added on `Service` controllers:
 
-`dns.alpha.kubernetes.io/external` creates a Route53 A record with `public` IPs of all the nodes
-`dns.alpha.kubernetes.io/internal` creates a Route53 A record with `private` IPs of all the nodes
+* `dns.alpha.kubernetes.io/external` creates a Route53 A record with
```
Comment (rk295) [inline dns-controller/README.md]: Happy to change the wording to reflect that.

File: dns-controller/README.md [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+we have a 1:1 mapping from an `etcd` node to a DNS name.)
 
-The dns-controller recognizes annotations on nodes.
+However, none of the nodes can reach the api server to register.  Nor 
+can end-users reach the API.  In future we might expose the API server 
+as a normal service via `Type=LoadBalancer` or via a normal Ingress, 
+but for now we just expose it via DNS.
 
-`dns.alpha.kubernetes.io/external` will set up records for accessing the resource externally
+The dns-controller recognizes annotations on nodes.
```
Comment (rk295) [inline dns-controller/README.md]: I'm unsure either tbh, I only slightly improved the formatting in this file, I can't vouch for its accuracy.

File: dns-controller/README.md [hunk 3]
```diff
@@ -1,22 +1,29 @@
...
+  the resource internally
 
 When added on `Service` controllers:
 
-`dns.alpha.kubernetes.io/external` creates a Route53 A record with `public` IPs of all the nodes
-`dns.alpha.kubernetes.io/internal` creates a Route53 A record with `private` IPs of all the nodes
+* `dns.alpha.kubernetes.io/external` creates a Route53 A record with 
+  `public` IPs of all the nodes
+* `dns.alpha.kubernetes.io/internal` creates a Route53 A record with 
+  `private` IPs of all the nodes
```
Comment (rk295) [inline dns-controller/README.md]: Ok, I'll re-word to explain that.

**Comment by justinsb** [comment]
Generally LGTM - if you clean up some of the things @sethpollack has observed we can merge IMO!

Thanks @rk295

**Comment by chrislovecnm** [comment]
What is the status on this?

**Comment by justinsb** [comment]
Going to merge & we can iterate, unless I hear an objection from @sethpollack

**Comment by sethpollack** [comment]
Go for it.

**Comment by justinsb** [comment]
Thanks @rk295 and @sethpollack

**Pull request merged**